packages/mdc-textfield/index.js

```diff
@@ -18,11 +18,12 @@
 import MDCComponent from '@material/base/component';
 import {MDCRipple} from '@material/ripple';
 
+/* eslint-disable no-unused-vars */
```

Comment:
Same as above. Could be removed to ensure unused var check is only removed for specific lines